lerna.json

```diff
@@ -1,11 +1,11 @@
 {
-  "capacitorVersion": "5.7.2",
+  "capacitorVersion": "6.0.0",
   "iosMinVersion": "13.0",
   "androidMinSdk": "22",
   "rnMinVersion": "0.63.4",
   "androidLiveUpdatesVersion": "0.5.0",
   "rnVersion": "0.5.1",
-  "iosVersion": "0.10.0",
-  "androidVersion": "0.9.0",
+  "iosVersion": "0.11.0",
+  "androidVersion": "0.10.0",
   "version": "0.9.0"
```

Comment:
We'll probably want to go ahead and change the version to 0.10.0 here to avoid having to open another PR